hbase2/README.md

```diff
@@ -15,25 +15,17 @@ permissions and limitations under the License. See accompanying
 LICENSE file.
 -->
 
-# HBase (0.98.x) Driver for YCSB
-This driver is a binding for the YCSB facilities to operate against a HBase 0.98.x Server cluster.
-To run against an HBase >= 1.0 cluster, use the `hbase10` binding.
+# HBase (2.y) Driver for YCSB
```

Comment:
Is there a reason not to consolidate the 3 different readmes into a single file?

Reply:
the way packaging works, we create a tarball of binaries for each binding and that packaging gets the README associated with that binding. We could restructure the hbase1, hbase2, and asynchbase modules to try to change this, but that's going to be a bunch of work.